pyne/mesh.py

```diff
@@ -583,6 +583,43 @@ def expand(self):
             self.mesh.mesh.tag_set_data(tag, list(self.mesh.iter_ve()), data)
 
 
+    def mult(self, multiplier):
+        if isinstance(multiplier, NativeMeshTag):
+            if self.size == 1:
+                if multiplier.size == 1:
+                    try:
+                        return self[:] * multiplier[:]
+                    except:
+                        print("Scalar data must have the same shape")
+                        #raise e 
+                elif multiplier.size > 1:
+                    try:
+                        return np.asarray([self[:]]) * multiplier[:]
```

Comment:
Would `self` have a shape of `(size,  voxels)` and multiplier `(voxels, size)`?  From our last conversation, I think we need  `self` to have `(voxels, size)` to have the correct multiplication. See also my comment in your second assert test